.github/scripts/build.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+WORKSPACE_ROOT=$1
+BUILDER_IMAGE=$2
+UBUNTU_BUILD_VER=$3
+
+set -e
+
+if [ -z "$WORKSPACE_ROOT" ]; then
+  echo "Error: WORKSPACE_ROOT is not set."
+  exit 1
+fi
+
+if [ -z "$BUILDER_IMAGE" ]; then
+  echo "Error: BUILDER_IMAGE is not set."
+  exit 1
+fi
+
+if [ -z "$UBUNTU_BUILD_VER" ]; then
+  echo "Error: UBUNTU_BUILD_VER is not set."
+  exit 1
+fi
+
+cd ${WORKSPACE_ROOT}
+git submodule update --init --recursive
+export CCACHE_DIR=${WORKSPACE_ROOT}/.caches/ccache
+export SCCACHE_DIR=${WORKSPACE_ROOT}/.caches/sccache
+export CCACHE_CONFIGPATH=${WORKSPACE_ROOT}/ccache.conf
+echo max_size = 600M >${WORKSPACE_ROOT}/ccache.conf
+echo log_file = ${WORKSPACE_ROOT}/ccache.log >>${WORKSPACE_ROOT}/ccache.conf
+export SCCACHE_CACHE_SIZE=200M
+export RUSTC_WRAPPER=sccache
+export DOCKER="docker run --rm \
```

Comment:
I don't think `DOCKER` or `DOCKER_ROOT` need to be exported.

Reply:
Done